main.js

```diff
@@ -0,0 +1,26 @@
+
+var $car = document.querySelector('.car');
+
+var directions = ['left', 'right', 'up', 'down'];
+var currentDirection = 0;
+
+document.addEventListener('keydown', function(e) {
+  // console.log('keycode: ', e.keyCode);
+  // console.log('code: ', e.code);
+
+  if(e.code === 'KeyA') {
+    currentDirection -= 45;
+
+    $car.setAttribute('style', ' width: 100px; transform-origin: center; transform: rotate(' +
```

Comment:
Instead of doing all this as inline styling, you should make CSS classes for the car, and each direction :+1: